scripts/sdk/src/deepbook.ts

```diff
@@ -4,22 +4,21 @@ import { SUI_CLOCK_OBJECT_ID, normalizeSuiAddress } from "@mysten/sui.js/utils";
 import { SuiClient, getFullnodeUrl } from "@mysten/sui.js/client";
 import { bcs } from "@mysten/sui.js/bcs";
 import {
-    ENV, DEEPBOOK_PACKAGE_ID, REGISTRY_ID, DEEP_TREASURY_ID, MY_ADDRESS,
+    DEEPBOOK_PACKAGE_ID, REGISTRY_ID, DEEP_TREASURY_ID, 
     OrderType, SelfMatchingOptions,
-    MANAGER_ADDRESSES
-} from './coinConstants';
-import { generateProof } from "./balanceManager";
+} from './config';
+import { BalanceManager, generateProof } from "./balanceManager";
 import { CoinKey, DEEP_SCALAR, FLOAT_SCALAR, GAS_BUDGET, Pool } from "./config";
 
-const client = new SuiClient({ url: getFullnodeUrl(ENV) });
+const client = new SuiClient({ url: getFullnodeUrl(process.env.ENVIRONMENT as "mainnet" | "testnet" | "devnet" | "localnet") });
```

Comment:
Can we default to testnet if env is not setup? Then default to mainnet after launch